src/plugins/dashboard/public/dashboard_top_nav/internal_dashboard_top_nav.tsx

```diff
@@ -313,10 +328,70 @@ export function InternalDashboardTopNav({
       });
     }
     if (showWriteControls && managed) {
-      allBadges.push(getManagedContentBadge(dashboardManagedBadge.getTooltip()));
+      const renderCustomBadge = () => {
+        const badgeButton = (
+          // @ts-ignore - EuiBadge href type issue
+          <EuiBadge
+            {...getManagedContentBadge(dashboardManagedBadge.getText(), false)}
+            onClick={() => setIsPopoverOpen(!isPopoverOpen)}
+          >
+            {managedText}
+          </EuiBadge>
```

Comment:
You should be able to satisfy the types without a `@ts-ignore` here - checkout the [badge with `onClick` docs](https://github.com/elastic/kibana/pull/190185) to see the things you should provide

Reply:
Thank you!